how-tos/rails/elasticsearch/readme.md

```diff
@@ -0,0 +1,31 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10423298/ea833a68-7079-11e5-84f8-0a925ab96893.png" width="60">  Elasicsearch Setup and Configuration
+
+<img src="https://cloud.githubusercontent.com/assets/1329385/12210263/7e707438-b60d-11e5-86a4-da08c62efd29.gif">
+
+## What is Elasticsearch (ES)?
+
+> Elasticsearch is a search server based on Lucene. It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License. Elasticsearch is the second most popular enterprise search engine after Apache Solr, also based on Lucene <a href="https://en.wikipedia.org/wiki/Elasticsearch" target="_blank">Wiki</a>
+
+We'll go over this description in class and focus on terms which may not be clear.
+
+<details>
+  <summary>Question to ask yourself, what is the difference between Elasticsearch and PostgreSQL?</summary>
```

Comment:
I will make this more clear, the link talks about the differences with RDMS in particular PostgreSQL but starts out about Mongo vs ES :/